.gitattributes

```diff
@@ -0,0 +1,7 @@
+* text=auto
+.travis.yml export-ignore
+.styleci.yml export-ignore
+.scrutinizer.yml export-ignore
+BACKERS.md export-ignore
+CONTRIBUTING.md export-ignore
+CHANGELOG.md export-ignore
```

Comment:
Do we have these files? or is this just a default? If we don't have them, can we remove them from here?

Reply:
Yes, they're defaults. Sure.

Reply:
I still think we don't have most of these files; have you maybe not pushed your latest code?